canela/bin/sj_tmole.py

```diff
@@ -0,0 +1,490 @@
+from canela import __version__
+import sys
+import os
+import shutil
+import subprocess
+import argparse
+import ase.io
+import ase.units
+from ase.utils import basestring
+from ase.calculators.turbomole import Turbomole
+
+
+def execute(args, input_str=None, error_test=True,
+            stdout_tofile=True):
+    """executes a turbomole executable and process the outputs"""
+
+    PIPE = subprocess.PIPE
+    Popen = subprocess.Popen

```

Comment:
use  `from subprocess import PIPE, Popen` instead

Reply:
Corrected import of variables and edited initial sj_tmole.py tracking commit.